common/controllers/framework/framework-overview.js

```diff
@@ -15,12 +15,18 @@ function frameworkOverview(req, res) {
   })
   const i18nContext = snakeCase(assessment.framework?.name || '')
 
+  const printable =
+    assessment.status === 'confirmed' ||
+    assessment.handover_occurred_at ||
+    (assessment.status === 'completed' && canAccess(`${i18nContext}:confirm`))
```

Comment:
Misplaced brackets? Should be `(['confirmed', 'completed'].includes(assessment.status) || assessment.handover_occurred_at) && canAccess(`${context}:print`)`

Reply:
This logic is actually the same as it was before:

- https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/8c8d8a65a6e927cf0d89e978b4828324642b19b5/common/presenters/move-to-identity-bar-actions/assessment-actions.js#L22
- `OR`
- https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/8c8d8a65a6e927cf0d89e978b4828324642b19b5/common/presenters/move-to-identity-bar-actions/assessment-actions.js#L63

I agree it's a little confusing though, which is why I pulled it out into a `printable` variable, but let me see if I can make it clearer.

Reply:
Although I do think we can get rid of the `canAccess`.

Reply:
Ah, ignore me, I didn't see that you're checking `&& canAccess(`${i18nContext}:print`)` just below :D

Reply:
I've decided your suggestion is better, I think it was actually the intention, I've pushed up an amended commit which changes it!